### PR TITLE
Query server time info

### PR DESCRIPTION
**Pull request by x22x22**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [X] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

add query server timestamp and timezone info

### CODE REVIEW

**Comment by wu-sheng** [comment]
Submodule doesn't update.

File: oap-server/server-query-plugin/query-graphql-plugin/src/main/java/org/apache/skywalking/oap/query/graphql/resolver/MetadataQuery.java [hunk 1]
```diff
@@ -93,4 +96,13 @@ public EndpointInfo getEndpointInfo(final int endpointId) throws IOException {
     public List<Database> getAllDatabases(final Duration duration) throws IOException {
         return getMetadataQueryService().getAllDatabases();
     }
+
+    public TimeInfo getTimeInfo() {
+        TimeInfo timeInfo = new TimeInfo();
+        SimpleDateFormat timezoneFormat = new SimpleDateFormat("ZZZZZZ");
+        Date date = new Date();
+        timeInfo.setCurrentTimestamp(date.getTime());
```
Comment (wu-sheng) [inline oap-server/server-query-plugin/query-graphql-plugin/src/main/java/org/apache/skywalking/oap/query/graphql/resolver/MetadataQuery.java]: Ref https://stackoverflow.com/questions/368094/system-currenttimemillis-vs-new-date-vs-calendar-getinstance-gettime

`System.currentTimeMillis() ` should be faster and good enough. Right?

**Comment by wu-sheng** [comment]
@kezhenxu94 Look like some rule causing this auto labeling failure.

File: oap-server/server-query-plugin/query-graphql-plugin/src/main/java/org/apache/skywalking/oap/query/graphql/resolver/MetadataQuery.java (hunk 1 above)
Comment (x22x22) [reply oap-server/server-query-plugin/query-graphql-plugin/src/main/java/org/apache/skywalking/oap/query/graphql/resolver/MetadataQuery.java]: how do get timezone?

Comment (wu-sheng) [reply oap-server/server-query-plugin/query-graphql-plugin/src/main/java/org/apache/skywalking/oap/query/graphql/resolver/MetadataQuery.java]: Sorry, I miss that.

**Pull request merged**